lighthouse-core/audits/installable-manifest.js

```diff
@@ -51,59 +138,82 @@ class InstallableManifest extends MultiCheckAudit {
   }
 
   /**
-   * @param {LH.Artifacts.ManifestValues} manifestValues
-   * @return {Array<string>}
+   * @param {LH.Artifacts} artifacts
+   * @return {Array<LH.IcuMessage>}
    */
-  static assessManifest(manifestValues) {
-    if (manifestValues.isParseFailure && manifestValues.parseFailureReason) {
-      return [manifestValues.parseFailureReason];
+  static getInstallabilityErrors(artifacts) {
+    const installabilityErrors = artifacts.InstallabilityErrors.errors;
+    const errorMessages = [];
+
+    // Filter out errorId 'in-incognito' since Lighthouse users are recommended to use incognito to test.
+    installabilityErrors.filter(err => err.errorId !== 'in-incognito');
```

Comment:
Filter returns a new array, it doesn't modify an existing array. probably best to `continue` this case in the for loop L151